docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,358 @@
+---
+title: .NET Aspire 9.0 (Release Candidate 1)
+description: Learn what's new with .NET Aspire 9.0 (Release Candidate 1).
+ms.date: 10/11/2024
+---
+
+# .NET Aspire 9.0 (Release Candidate 1)
+
+.NET Aspire 9.0 is the next major release of .NET Aspire; it will support BOTH .NET 8 (LTS) and .NET 9. .NET Aspire 9 addresses some of the most highly requested features and pain points from the community. 
```

Comment:
More of a question, do we normally refer to `.NET Aspire 9.0` or `.NET Aspire 9`?

Reply:
pain points -> feedback?

Reply:
Suggest edits